subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/ArtifactDependenciesIntegrationTest.groovy

```diff
@@ -350,10 +348,16 @@ dependencies {
     compile "org.gradle.test:lib:1.0@zip"
     compile "org.gradle.test:dist:1.0"
 }
+
+// Resolve outside of TaskAction to work with Configuration Cache
```

Comment:
I think this comment is unnecessary. Configuration cache compatibility is pretty much the norm in all of our integration tests. If we commented this in each place we had to make integration test changes for CC compatibility, we'd have this comment everywhere